pkg/operator/certrotation/signer_test.go

```diff
@@ -319,9 +325,16 @@ func TestEnsureSigningCertKeyPair(t *testing.T) {
 		{
 			name: "update no annotations",
 			initialSecret: &corev1.Secret{
-				ObjectMeta: metav1.ObjectMeta{Namespace: "ns", Name: "signer", ResourceVersion: "10"},
-				Type:       corev1.SecretTypeTLS,
-				Data:       map[string][]byte{"tls.crt": {}, "tls.key": {}},
+				ObjectMeta: metav1.ObjectMeta{Namespace: "ns", Name: "signer",
+					ResourceVersion: "10",
+					Annotations: map[string]string{
```

Comment:
why the change, the scenario says `update no annotations`

Reply:
hmm, I should rename this to "no necessary signer annotations"

Reply:
Done